src/Input/Input.spec.js

```diff
@@ -207,5 +208,16 @@ describe('<Input />', () => {
         assert.strictEqual(wrapper.hasClass(classes.error), true);
       });
     });
+
+    describe('required', () => {
+      beforeEach(() => {
+        setFormControlContext({ required: true });
```

Comment:
I guess we can move that into the `it` block. But it depend what's we anticipate, your call.